### ISSUE TITLE
Unable to run accumulo

### ISSUE THREAD

**Issue by mindscratch** [apache/accumulo-docker#4]
I built the image and ran `docker run accumulo` and I got:

```
HADOOP_HOME=/path/to/hadoop is not set to a valid directory in accumulo-env.sh
```

I added `HADOOP_HOME` to `Dockerfile` and then got:

```
2018-09-10 21:25:49,371 [start.Main] ERROR: Unable to find Hadoop Configuration class on classpath, check configuration.
java.lang.ClassNotFoundException: org.apache.hadoop.conf.Configuration
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at org.apache.accumulo.start.classloader.AccumuloClassLoader$1.loadClass(AccumuloClassLoader.java:198)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at org.apache.accumulo.start.Main.main(Main.java:52)
```

I added `ENV HADOOP_CONF_DIR $HADOOP_HOME/etc/hadoop` and I still get `Unable to find Hadoop Configuration class on classpath, check configuration.`.

**Comment by mikewalch** [comment]
This docker image is based off Accumulo 2.0.0-SNAPSHOT which has changed:
* 2.0.0-SNAPSHOT requires Hadoop 3.x
* 2.0.0-SNAPSHOT expects accumulo.properties format rather than accumulo-site.xml

I created pull request #5 that should fix these issues.  I did not have time to test it yet.  I will try to test it tomorrow.  Let me know if it works for you.

**Comment by mindscratch** [comment]
That works! I ran `docker run accumulo miniCluster` ...how would I access the accumulo UI?

**Comment by mikewalch** [comment]
Great! I don't think Accumulo minicluster starts the Accumulo Monitor/UI but I could be wrong.  You might need to start each Accumulo process individually using Docker.  If you want to look run Accumulo locally for development, check out [Uno](https://github.com/apache/fluo-uno). Many Accumulo developers use Uno for local development and testing.  There is also [Muchos](https://github.com/apache/fluo-muchos) for starting an Accumulo cluster on EC2.

**Comment by ctubbsii** [comment]
Fixed by #5
